src/CommunityToolkit/Xamarin.CommunityToolkit/Views/HexLayout/HexLayout.shared.cs

```diff
@@ -0,0 +1,266 @@
+﻿using System;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.UI.Views
+{
+    /// <summary>
+    /// A Layout that arranges the elements in a honeycomb pattern.
+    /// Based on https://github.com/AlexanderSharykin/HexGrid. Created by Alexander Sharykin.
+    /// </summary>
+    public class HexLayout : Layout<View>
+    {
+        public static readonly BindableProperty OrientationProperty =
+         BindableProperty.Create(nameof(Orientation), typeof(HexOrientation), typeof(HexLayout), HexOrientation.Vertical,
+             BindingMode.TwoWay, propertyChanged: (bindable, oldvalue, newvalue) => ((HexLayout)bindable).InvalidateMeasure());
```

Comment:
Can we refactor that property changed to a method?

Reply:
Changed.